docs/examples/affinity/cookie/README.md

```diff
@@ -1,28 +1,29 @@
-# Sticky Session
+# Sticky sessions
 
-This example demonstrates how to achieve session affinity using cookies
+This example demonstrates how to achieve session affinity using cookies.
 
 ## Deployment
 
-Session stickiness is achieved through 3 annotations on the Ingress, as shown in the [example](ingress.yaml).
+Session stickiness can be configured using the following annotations. Make sure to prefix their names with `nginx.ingress.kubernetes.io/` when setting them.
 
-|Name|Description|Values|
+|Name <small>without prefix</small>|Description|Value|
 | --- | --- | --- |
-|nginx.ingress.kubernetes.io/affinity|Sets the affinity type|string (in NGINX only ``cookie`` is possible|
-|nginx.ingress.kubernetes.io/session-cookie-name|Name of the cookie that will be used|string (default to INGRESSCOOKIE)|
-|nginx.ingress.kubernetes.io/session-cookie-hash|Type of hash that will be used in cookie value|sha1/md5/index|
-|nginx.ingress.kubernetes.io/session-cookie-expires|The value is a date as UNIX timestamp that the cookie will expire on, it corresponds to cookie Expires directive|number of seconds|
-|nginx.ingress.kubernetes.io/session-cookie-max-age|Number of seconds until the cookie expires that will correspond to cookie `Max-Age` directive|number of seconds|
+|affinity|Type of the affinity, used to enable it|string (NGINX only supports `cookie`)|
+|session-cookie-name|Name of the cookie that will be created|string (defaults to `INGRESSCOOKIE`)|
+|session-cookie-hash|Type of hash that will be used to generate the cookie value|`sha1`/`md5`/`index` (defaults to `sha1`)|
+|session-cookie-path|Path that will be set on the cookie (required if your [Ingress paths][ingress-paths] use regular expressions)|string (defaults to the currently [matched path][ingress-paths])|
+|session-cookie-expires|Date the cookie will expire on, corresponds to the `Expires` cookie directive|UNIX timestamp|
```

Comment:
Now that we are on this, lets fix more :) The expected value of this field is also number of seconds, not UNIX timestamp. To give you some context, actual value of `Expires` cookie is calculated by adding this number of seconds to the current Unix time when a request occur: https://github.com/kubernetes/ingress-nginx/blob/79c52cf09417af61f4c1d067bd48304f9b3fe743/rootfs/etc/nginx/lua/balancer/sticky.lua#L66

Reply:
What's the difference to `max-age` then when they are both relative to the current time?

Reply:
I think I figured it out. Could you take another look, @ElvinEfendi?

Reply:
@shroudedcode both `Max-Age` and `Expires` serve for the same purpose - to limit the life of a cookie. Older browsers do not support `Max-Age` and `Max-Age` takes precedence when both given.

The main difference is their value. `Max-Age` expects a duration while `Expires` expects a fixed datetime. However the ingress-nginx annotations expects duration for both of them (it then internally adds that duration to current time when setting `Expires` value).

`Expires` is very limited and vulnerable to time drift between server and client and at least to my understand usually is used to keep support for older browsers that do not support `Max-Age`.